CODEOWNERS

```diff
@@ -1 +0,0 @@
-*    @Pliner @zidad @micdenny @luigiberrettini
```

Comment:
See https://github.com/dotnet/runtime/blob/main/.github/CODEOWNERS 

File is placed in .github directory.